### DISCUSSION TITLE
Arabic Letter

### DISCUSSION THREAD

**Discussion by farahelmasri** [IBM/plex#553] [Q&A]
hello
i have an issue with 1 of the arabic letters that is missing, what is the best way forward to get this letter created?

The arabic letter K needs to be adjusted to follow the same curve as L in red.

otherwise it looks like its a broken letter or the end of it is cut

![292770168-4fa49362-36d4-40e7-9d49-820bef35ef7a](https://github.com/IBM/plex/assets/154777706/fb911a51-2c72-4f4a-8da8-0c594b8d48d1)

**Comment by BoldMonday** [comment]
Closing: duplicate of https://github.com/IBM/plex/issues/550

**Reply by farahelmasri** [reply]
Do we have any other font recommendations that has the letter as per what we wanted?
